feature-libs/product-configurator/rulebased/components/price/configurator-price.component.ts

```diff
@@ -31,7 +31,11 @@ export class ConfiguratorPriceComponent {
     if (this.formula.priceTotal) {
       return this.priceTotal;
     } else {
-      return '+ ' + this.formula.price?.formattedValue;
+      if (this.formula.price?.value && this.formula.price?.value >= 0) {
```

Comment:
I think the condition should be `this.formula.priceTotal.value >= 1` and not `this.formula.priceTotal.value >= 0`.
The end-user wants to see price when it is cheaper or more expensive but not when the price equals zero.
Plz, add a test for zero case.

Reply:
same as above, prefer to go for 
this.formula.priceTotal.value > 0